migrator/migrations/n_52_to_n_53_postgres_simple_access_scopes/migration.go

```diff
@@ -31,20 +46,70 @@ var (
 				return errors.Wrap(err,
 					"moving simple_access_scopes from rocksdb to postgres")
 			}
+			legacyPermissionSetStore, err := legacypermissionsets.New(databases.PkgRocksDB)
+			if err != nil {
+				return err
+			}
+			if err := movePermissionSets(databases.GormDB, databases.PostgresDB, legacyPermissionSetStore); err != nil {
+				return errors.Wrap(err,
+					"moving permission_sets from rocksdb to postgres")
+			}
+			legacyRoleStore, err := legacyroles.New(databases.PkgRocksDB)
+			if err != nil {
+				return err
+			}
+			if err := moveRoles(databases.GormDB, databases.PostgresDB, legacyRoleStore); err != nil {
+				return errors.Wrap(err,
+					"moving roles from rocksdb to postgres")
+			}
 			return nil
 		},
 	}
 	batchSize = 10000
 	schema    = pkgSchema.SimpleAccessScopesSchema
 	log       = loghelper.LogWrapper{}
+
+	accessScopeIDMapping = map[string]string{
+		"denyall":      "ffffffff-ffff-fff4-f5ff-fffffffffffe",
+		"unrestricted": "ffffffff-ffff-fff4-f5ff-ffffffffffff",
+	}
+
+	permissionsetIDMapping = map[string]string{
+		"admin":                 "ffffffff-ffff-fff4-f5ff-ffffffffffff",
+		"analyst":               "ffffffff-ffff-fff4-f5ff-fffffffffffe",
+		"continuousintegration": "ffffffff-ffff-fff4-f5ff-fffffffffffd",
+		"none":                  "ffffffff-ffff-fff4-f5ff-fffffffffffc",
+		"scopemanager":          "ffffffff-ffff-fff4-f5ff-fffffffffffb",
+		"sensorcreator":         "ffffffff-ffff-fff4-f5ff-fffffffffffa",
+		"vulnmgmtapprover":      "ffffffff-ffff-fff4-f5ff-fffffffffff9",
+		"vulnmgmtrequester":     "ffffffff-ffff-fff4-f5ff-fffffffffff8",
+		"vulnreporter":          "ffffffff-ffff-fff4-f5ff-fffffffffff7",
+	}
 )
 
+func convertAccessScopeID(accessScopeID string) string {
+	identifierSuffix := strings.TrimPrefix(accessScopeID, accessScopeIDPrefix)
+	replacement, found := accessScopeIDMapping[identifierSuffix]
+	if found {
+		return replacement
+	}
+	_, err := uuid.FromString(identifierSuffix)
+	if err != nil {
+		generatedID := uuid.NewV4().String()
```

Comment:
Can we use `GenerateAccessScopeID` here?

Do we need to keep old UUIDs? If not, then we can remove all this `uuid.FromString(identifierSuffix)` and always generate new V4 UUIDs.

The same can be applied to `convertPermissionSetID`.

Reply:
Technically, it would be doable. There still would have to be an exception for the default permission sets and access scopes, which get overwritten at each start from central.
I suppose the number of access scopes, permission sets and roles should stay reasonably low, but I'd still would prefer to avoid adding resource overhead for the migration. I would also like to keep the adjustment effort as low as possible for the case customers would have stored some identifiers in scripts and would have to adjust the list after the migration.